HTML/Forget Maze Not.html

```diff
@@ -17,25 +18,25 @@
                 <span class="page-header-section-title">Forget Maze Not</span>
             </div>
             <div class="page-content">
-                <img src="img/Component/Forget Maze Not.svg" class="diagram">
-                <h2>On the Subject of Forget Maze Not</h2>
-                <p class="flavour-text">flavor in get you I hope didn't lost this text!</p>
-                <ul>
-                    <li>This module will cycle through a few cells of a maze, showing the walls and coordinates of the cell.</li>
-                    <li>With every solve, ignoring some specific modules, the module will move to the next stage, revealing more cells. Put them together to form a maze.</li>
-                    <li>On some stages, a cell will be colored green. After all the non-ignored modules are solved, navigate to these cells in the order of their stage number and submit them by clicking the middle button.</li>
-                    <li>Any other colored cell are special cells and they are not part of the main maze. Look at the direction's below for the respective color. Click the middle button on a special cell to put the module in interactive mode, and press it again to exit interactive mode.
-                        <ul>
-                            <li>Red: Using the maze below, navigate the defuser to the coordinates on the bottom left. The current position is in the middle of the cell.</li>
-                            <li>Yellow: The coordinates in the middle represents a cell that was displayed on an earlier stage. Using the arrows, input the walls of that cell.</li>
-                            <li>Blue: Same as yellow, but instead of inputting the walls, input the stage number the cell was displayed on using the arrows.</li>
-                            <li>Magenta: The defuser will have to find the exit of a normal 5x5 maze on the module.</li>
-                            <li>Cyan: Same as magenta, but the maze is inverted (You can only walk through walls).</li>
-                        </ul>
-                    </li>
-                    <li>Moving, exiting interactive mode, and submitting cells incorrecty will lead to a strike.</li>
-                </ul>
-                <img src="img/Forget Maze Not/Maze.svg" width="300" height="300" style="display:block;margin-left:auto;margin-right:auto;margin-top:20px;">
+				<img src="img/Component/Forget Maze Not.svg" class="diagram">

```

Comment:
Be sure to grab the repo copy first before making changes. It was changed from tabs to spaces and you're trying to revert it.